### ISSUE TITLE
 Prisma Seed Script Fails After Upgrading to v5.16.0 (DATABASE_URL Error)

### ISSUE THREAD

**Issue by khylpe** [prisma/prisma#24658]
### Bug description

After upgrading to `@prisma/client@5.16.0` and `prisma@5.16.0`, my Prisma seed script located in the `/prisma` folder stopped working. The script is executed using a simple command in my `package.json`:

```
"scripts": {
    ...,
    "seed_defaultData": "ts-node prisma/myFile.js"
},
```

When I run the script (`npm run seed_defaultData`), I now get the following error:

`PrismaClientInitializationError: error: Environment variable not found: DATABASE_URL.`.

Important Details:

- No Code Changes: I haven't made any changes to the seed script itself, my Prisma schema (schema.prisma), or my environment variables.
- Downgrading Works: Reverting back to `@prisma/client@5.15.0` and `prisma@5.15.0` resolves the issue and the seed script runs successfully.
- Possible Cause: I'm aware that Prisma v5.16.0 introduced changes related to the handling of the `/prisma` folder. Could this be causing the `DATABASE_URL` error in my seed script?

### How to reproduce

1. Install @prisma/client@5.16.0 and prisma@5.16.0.
2. Place a seed script (e.g., myFile.js) in the /prisma folder.
3. Define a script in package.json to execute the seed script.
4. Run the script using npm run <script_name>.

### Expected behavior

The seed script should run successfully, populating the database with data.

### Prisma information

```prisma

generator client {
       provider = "prisma-client-js"
}

datasource db {
       provider = "mysql"
       url      = env("DATABASE_URL")
}

model Language {
       id     Int      @id @default(autoincrement())
       name   String   @unique
       Cursus Cursus[]

       @@map("language")
}

// other models

```

seed script :
```ts

const { PrismaClient } = require('@prisma/client');

const prismaTest = new PrismaClient();

async function main() {
       const defaultLanguages = [
              { name: 'french' },
              { name: 'english' },
       ];

       // Start a transaction
       const transaction = await prismaTest.$transaction(
              async (prisma) => {
                     // Upsert languages
                     for (let i = 0; i < defaultLanguages.length; i++) {
                            const language = defaultLanguages[i];
                            await prisma.language.upsert({
                                   where: { name: language.name },
                                   update: {},
                                   create: language,
                            });
                     }

                     // Get the IDs of the 'french' and 'english' languages
                     const frenchLanguage = await prisma.language.findUnique({
                            where: { name: 'french' },
                     });

                     const englishLanguage = await prisma.language.findUnique({
                            where: { name: 'english' },
                     });

                     if (!frenchLanguage || !englishLanguage) {
                            throw new Error('Failed to find French or English language.');
                     }
      // Other private code

              }, {
              maxWait: 5000, // default: 2000
              timeout: 10000, // default: 5000
       });
}

main()
       .catch((e) => {
              console.error(e);
              process.exit(1);
       })
       .finally(async () => {
              await prismaTest.$disconnect();
       });

```

### Environment & setup

- OS: Windows
- Database: MySQL
- Node.js version: v20.9.0

### Prisma Version

```
prisma@5.16.0
@prisma/client@5.16.0
```

**Comment by aryamohanan** [comment]
We are also experiencing the same error after upgrading the version to 5.16.0.

> PrismaClientInitializationError: error: Environment variable not found: DATABASE_URL

**Comment by romain-sen** [comment]
Same error here with the environment variables : we are missing all our environment variables provided by Nest.js coming from .env file.

**Comment by khylpe** [comment]
An issue about it was recently closed  : https://github.com/prisma/prisma/issues/24634
Here is the fix : https://github.com/prisma/prisma/pull/24649

Closing this issue since the patch should be released soon in `5.16.1`

**Comment by jkomyno** [comment]
Hello @khylpe, @romain-sen. and @aryamohanan, we've just released [Prisma 5.16.1](https://www.npmjs.com/package/prisma/v/5.16.1), which contains the fix to this regression (and an additional test to ensure it doesn't happen again).

Please install it, re-run `prisma generate`, and you should be good to go. You're welcome to leave feedback here in the comments :)

**Comment by khylpe** [comment]
@jkomyno Thank a lot for the fix. Worked perfectly :)
